npm-update

```diff
@@ -59,6 +59,8 @@ def write_package_json(package_json_path, data):
 def prefix(name, version):
     if not version[0].isdigit():
         return version
+    if name.startswith("@patternfly"):
+        return "prerelease"
```

Comment:
Is that always guaranteed to exist *and* be newer than the latest release? I.e. could it happen that when they do a new major release, the prerelease is still the older one?

Reply:
It is something that is manually done - you need to tag it with `prerelease`. But pretty sure that PF team does this with script and that we can count on that.